exercises/rollback-with-saga/README.md

```diff
@@ -1,8 +1,75 @@
-## Exercise #3: Implementing a Rollback Action with the Saga Pattern
+## Exercise #3: Rollback with the Saga Pattern
 
 During this exercise, you will:
 
-Make your changes to the code in the `practice` subdirectory (look for
-`TODO` comments that will guide you to where you should make changes to
-the code). If you need a hint or want to verify your changes, look at
-the complete version in the `solution` subdirectory.
\ No newline at end of file
+- Orchestrate Activities using a Saga pattern to implement compensating transactions
+- Handle failures with rollback logic
+
+Make your changes to the code in the `practice` subdirectory (look for `TODO`
+comments that will guide you to where you should make changes to the code). If
+you need a hint or want to verify your changes, look at the complete version in
+the `solution` subdirectory.
+
+## Part A: Review your new rollback Activities
+
+This Exercise uses the same structure, and the same Error, as in the previous
+Exercises — meaning that it will fail at the very end on `ProcessCreditCard` if
+you provide it with a bad credit card number.
+
+Three new Activities have been created to demonstrate rollback actions.
+`UpdateInventory` is a new step that would run normally (whether or not the
+Workflow encounters an error). `RevertInventory` has also been added as a
+compensating action for `UpdateInventory`. Finally, `RefundCustomer` has been
+added as a compensating action for `SendBill`.
+
+1. Review these new Activities at the end of the `activities.go` file. None of
+   them make actual inventory or billing changes, because the intent of this
+   Activity is to show Temporal features, but you should be able to see where
+   you could add functionality here.
+2. Close the file.
+
+## Part B: Add your new rollback Activities to your Workflow
+
+1. Open `workflow.go`. Note that the `UpdateInventory` Activity to your Workflow
```

Comment:
This feels incomplete?

Reply:
Fixed this in a commit yesterday, good catch.